share/qtcreator/sfdk/modules/35-building-sdk-assistant.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": 1,
+    "domain": "building",
+    "trBrief": "Building packages",
+
+    "trDescription": "MANAGING SDK BUILD TOOLS\n\nThe Sailfish SDK abstracts the support for cross-development for particular target HW architectures and Sailfish OS versions (forward compatibility applies) in form of add-on SDK Build Tools.\n\nTwo types of SDK Build Tools are recognized:\n\n1. SDK Build Target - a target-compatible Sailfish OS image, and\n2. SDK Build Tooling - a host-compatible collection of build time dependencies.\n\nThe shorter terms \"build tools\", \"(build) tooling(s)\" and \"(build) target(s)\" are used instead of the official terms where the context admits. Also, since a single SDK Build Tooling is usually used with multiple SDK Build Targets that enable development for the same Sailfish OS version and differ only by the supported target architecture, the term SDK Build Tools is sometimes used to denote a particular combination of SDK Build Target and Tooling, uniquely identified by the SDK Build Target name.\n\nThe prefered way of managing SDK Build Tools is through the SDK Installer/Maintenance Tool (see the 'maintain' command). Custom build tools can be added using the 'tools' command.",
+
+    "worker":
+    {
+        "type": "engine-command",
+        "version": 1,
+        "program": "sdk-assistant",
+        "initialArguments": ["--non-interactive"],
+        "omitSubcommand": true
+    },
+
+    "commands":
+    [
+        {
+            "name": "tools",
+            "trSynopsis": "[tooling|target] create <name> {<URL>|<file>}\n[tooling|target] update <name>\n[tooling|target] remove <name>\n[tooling|target] list [--slow]",
```

Comment:
It would be nice to tell what "--slow" does

Reply:
Fixed, thanks